### DISCUSSION TITLE
CarouselView on Windows?

### DISCUSSION THREAD

**Discussion by plettb** [dotnet/maui#6837] [Q&A]
Does anyone have CarouselView working on Windows?  I'd love to see working code demonstrating it!

**Comment by IPdotSetAF** [comment]
It looks so bad on windows, I had to replace it with a collection view for Windows platform.

**Comment by jbfranklin** [comment]
It looks like shit I agree.

**Comment by Brosten** [comment]
Seems to be lot's of trouble around the carouselview. At least on windows. Hope they have time to fix the worst issues soon. 
Two main issues for me
- Mousewheel scrolling between the items often lags and the UI stay between items
- Flickers on resize
- Adding items to the boud collection does show up in the indicator view, but you cand scroll to them
- Would like to be able to "swipe" with mouse-pointer and not only using scroll-wheel.

**Comment by Xyncgas** [comment]
CarouselView was but a nice attempt
I say use Unity engine to develop APP now on

**Comment by BeepBeepBopBop** [comment]
I don't have it working well on Android or iOS either. It's much closer to something usable than Windows though.
